spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-appactive/src/main/java/com/alibaba/cloud/appactive/consumer/WebClientInterceptor.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2013-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.cloud.appactive.consumer;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.alibaba.cloud.appactive.common.ModifyHttpServletRequestWrapper;
+import com.alibaba.cloud.appactive.common.UriContext;
+import com.alibaba.cloud.appactive.constant.AppactiveConstants;
+import io.appactive.java.api.base.AppContextClient;
+import io.appactive.support.log.LogUtil;
+import org.slf4j.Logger;
+
+import org.springframework.lang.NonNull;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+/**
+ * @author yuluo
+ */
+public class WebClientInterceptor extends HandlerInterceptorAdapter {

```

Comment:
Is the interceptor interface of WebClient `ExchangeFilterFunction`?

Reply:
sorry, I understood it into a web request, give me a little time and I finish it.